plexupdate.sh

```diff
@@ -119,21 +118,16 @@ else
 	fi
 fi
 
-if [ ! "${RELEASE}" = "" ]; then
-	echo "ERROR: RELEASE keyword is deprecated and should be removed from .plexupdate" >&2
-	echo "       Use DISTRO and BUILD instead to manually select what to install (check README.md)" >&2
-	exit 255
-fi
-
```

Comment:
I'm a bit concerned that removing this already may cause other end-users to get frustrated before they figure out what's going on.